lib/api/controllers/collectionController.js

```diff
@@ -330,6 +333,7 @@ class CollectionController extends BaseController {
   create(request) {
     assertHasIndex(request);
 
+    // TODO indexCache actions here
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)